node-notes-cli/app.js

```diff
@@ -0,0 +1,13 @@
+import readNotes from './read.js';
+import addNote from './add.js';
+import deleteNote from './delete.js';
+import updateNote from './update.js';
+
+const [,, feature, ...args] = process.argv;
+switch (feature) {
+  case 'read': readNotes(1); break;
+  case 'add': addNote(args); break;
+  case 'delete': deleteNote(args); break;
+  case 'update': updateNote(args[0], args.slice(1)); break;
```

Comment:
These are all async functions, so to properly handle errors from them you need to `await` them. You also need to wrap them in try/catch (you can wrap the entire `switch`).